src/MLabsPAB/CardanoCLI.hs

```diff
@@ -181,7 +181,7 @@ buildTx ::
 buildTx pabConf ownPkh buildMode tx =
   callCommand @w $ ShellArgs "cardano-cli" opts (const ())
   where
-    ownAddr = Ledger.pubKeyHashAddress ownPkh
+    ownAddr = Ledger.pubKeyHashAddress (Ledger.PaymentPubKeyHash ownPkh) Nothing
```

Comment:
Any way we can get the full address here? Maybe have it passed in?
Ideally we use full addresses everywhere we can, and always keep those staking addresses around

Reply:
Oddly enough, afaik the PAB doesn't support this, it can only return the ownPkh, but we could work around this by putting the address itself in the PABConfig

Reply:
Yeah that sucks, we'll just have to make sure to use non-staking addresses for this project